src/features/world/scenes/Kingdom.ts

```diff
@@ -2,10 +2,58 @@ import mapJSON from "assets/map/kingdom.json";
 
 import { SceneId } from "../mmoMachine";
 import { BaseScene } from "./BaseScene";
+import { NPCName, NPC_WEARABLES } from "lib/npcs";
+import { FactionName } from "features/game/types/game";
+import { BumpkinContainer } from "../containers/BumpkinContainer";
+import { interactableModalManager } from "../ui/InteractableModals";
+
+export type FactionNPC = {
+  npc: NPCName;
+  x: number;
+  y: number;
+  direction?: "left" | "right";
+  faction: Omit<FactionName, "nightshades">;
+};
+
+export const KINGDOM_NPCS: FactionNPC[] = [
+  {
```

Comment:
Can we give them names as well?

Currently all of our NPCs have the Label above them